cql3/statements/alter_keyspace_statement.cc

```diff
@@ -87,7 +87,8 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
                 const std::map<sstring, sstring>& current_rfs = ks.metadata()->strategy_options();
                 for (const auto& [new_dc, new_rf] : _attrs->get_replication_options()) {
                     auto it = current_rfs.find(new_dc);
-                    if (it != current_rfs.end() && !validate_rf_difference(it->second, new_rf)) {
```

Comment:
` cql: ALTER tablets KS cannot change RF by >2 in new DCs `

should be `>=2` or `>1`. Both in this commit message and in PR description.

Reply:
Fixed, thanks.